packages/boot/src/boot.component.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/boot
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {BootStrapper} from './boot-strapper';
+import {Component, Application, CoreBindings} from '@loopback/core';
+import {inject, BindingScope} from '@loopback/context';
+import {ControllerBooter} from './booters';
+
+export class BootComponent implements Component {
```

Comment:
Do we still need this component?

Reply:
The Component binds the `BootStrapper` and default list of `Booter`'s

Reply:
There should be TSDocs on this class/ctor to make that clear, then. :)